### ISSUE TITLE
Could you help to publish an Ivy distribution

### ISSUE THREAD

**Issue by KevinZhang19870314** [code-farmz/ng-otp-input#84]
I am now upgrading my angular project to version 13, when do ng build, it has an warning:

```
- Generating browser application bundles (phase: setup)...
Processing legacy "View Engine" libraries:
- ng-otp-input [es2015/esm2015] (git+https://github.com/code-farmz/ng-otp-input.git)
Encourage the library authors to publish an Ivy distribution.
```

**Comment by aditbharadwaj** [comment]
Yes please we need a version update for angular 13

**Comment by code-farmz** [comment]
fixed in v1.8.3
